src/gui/gui.h

```diff
@@ -126,6 +126,8 @@ class DissolveWindow : public QMainWindow
     const int recentFileLimit_;
 
     private:
+    // Set the input file path
+    void setInputFilename(const QString filename);
```

Comment:
Tiny thing, but the Qt people like strings to be passed as const-references:
```suggestion
    void setInputFilename(const QString &filename);
```